tsfile/pom.xml

```diff
@@ -53,6 +53,11 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.jpountz.lz4</groupId>
+            <artifactId>lz4</artifactId>
+            <version>1.3.0</version>
+        </dependency>
```

Comment:
I think we should update the LICENSE-binary as well. :)

Reply:
Sure~ I will fix it~